### PR TITLE
Return code + summary

### PR DESCRIPTION
**Pull request by imobach**
* Return a proper error code after running the tests
* Show a summary of passed, failing a missing tests

tests-summary

### CODE REVIEW

File: package/rubygem-aytests.changes [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+- Return a proper error code after running the tests
+- Show a summary of passed, failing a missing tests
+- 1.0.3
+
+-------------------------------------------------------------------
 Mon Jan 11 12:55:49 UTC 2016 - mfilka@suse.com
 
 - fixed MAC formatting when passed as parameter to virtualbox
   during machine creation
-- 1.0.3
```
Comment (kobliha) [inline package/rubygem-aytests.changes]: Intentional?

**Comment by kobliha** [comment]
Looks great from my POV :) I'd say LGTM
(if the version is intentional)

File: lib/aytests/cli.rb [hunk 2]
```diff
@@ -40,11 +51,16 @@ def test(*files)
             provider: options["provider"] || ENV["AYTESTS_PROVIDER"] || :libvirt,
             headless: options[:headless] || ENV["AYTESTS_HEADLESS"] == "true"
           )
-          runner.run
+          # When a test fails, a non-zero return code will be returned
+          results[test_file] = runner.run ? :passed : :failed
         else
+          results[test_file] = :missing
           $stderr.puts "File #{test_file} does not exist"
```
Comment (mchf) [inline lib/aytests/cli.rb]: Sort of log file should be better from my POV. Searching for this reports on stderr is not comfortable (I know I can redirect output into a file but who cares when starting the test ;-)

File: package/rubygem-aytests.changes (hunk 1 above)
Comment (mchf) [inline package/rubygem-aytests.changes]: version looks weird. It should be 1.0.4 (+ update VERSION and spec) or stay with 1.0.3 and fix my double dash issue.

Comment (imobach) [inline package/rubygem-aytests.changes]: Yes, it's intentional. We didn't release 1.0.3 yesterday (I was waiting to get this fix merged).

**Comment by imobach** [comment]
Ok, the version was intentional but was wrong. We released 1.0.3 yesterday, so I'm bumping it to 1.0.4.

File: lib/aytests/cli.rb (hunk 2 above)
Comment (imobach) [inline lib/aytests/cli.rb]: I think you're right. Now we use the AYTests logger.

File: package/rubygem-aytests.changes [hunk 3]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Jan 12 08:36:25 UTC 2016 - igonzalezsosa@suse.com
+
+- Return a proper error code after running the tests
+- Show a summary of passed, failing a missing tests
```
Comment (mchf) [inline package/rubygem-aytests.changes]: ... and missing ...

**Comment by mchf** [comment]
LGTM

**Comment by imobach** [comment]
Thanks!

**Pull request merged**